libraries/provider_runit_service.rb

```diff
@@ -136,6 +140,15 @@ def configure_service
             else
               Chef::Log.debug("Control signals not specified for #{new_resource.service_name}, continuing")
             end
+
+            # Create/Delete service down file
+            # To prevent unexpected behavior, require users to explicitly set
+            # delete_downfile to remove any down file that may already exist
+            if new_resource.start_down
+              do_action(downfile, :create)
+            elsif new_resource.delete_downfile
```

Comment:
Hmm instead of having the delete_downfile boolean parameter could would automatically try and delete the down file if it exists?  I feel like the original repo critics would be skeptical about adding a specific parameter just to delete the down file.